homepage/homepage/app/(others)/examples/page.tsx

```diff
@@ -446,7 +439,7 @@ const reactExamples: Example[] = [
     description:
       "A game that shows how to manage state in a trusted environment through the experimental HTTP API.",
```

Comment:
We refer to this now as [JazzRPC](https://jazz.tools/docs/server-side/jazz-rpc), so it might be more consistent to use that wording here too to avoid confusion with the [token-based utilities](https://jazz.tools/docs/server-side/communicating-with-workers/http-requests). Should we update this yet, or should I pop in a separate PR for this once we drop the `experimental_` prefix?